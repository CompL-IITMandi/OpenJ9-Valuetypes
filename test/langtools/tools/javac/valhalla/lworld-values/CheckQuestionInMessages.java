/*
 * @test /nodynamiccopyright/
 * @bug 8222790
 * @summary javac diagnostics don't discriminate between inline types and there nullable projection types.
 *
 * @compile/fail/ref=CheckQuestionInMessages.out -XDrawDiagnostics CheckQuestionInMessages.java
 */

import java.util.List;

inline class X {
    List<X?> ls = new Object();    
    X?[] xa = new Object?[10];
}
