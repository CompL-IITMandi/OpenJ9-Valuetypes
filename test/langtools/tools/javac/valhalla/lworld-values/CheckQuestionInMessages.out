CheckQuestionInMessages.java:12:19: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: java.lang.Object, java.util.List<X?>)
CheckQuestionInMessages.java:13:15: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: java.lang.Object[], X?[])
2 errors
